ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v1/aemform/README.md

```diff
@@ -13,9 +13,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-AEM Forms Container (v1)
+Adaptive Forms Container (v1)
 ====
-AEM Forms Container component written in HTL.
+Adaptive Forms Container component written in HTL.
 
 ## Features
 * Acts as a container to use AEM Forms pages (ie) Adaptive Form or Interactive Communication inside an AEM Sites Page
```

Comment:
Needs review if we want to change AEM Forms to Adaptive Form in multiple locations in the PR.